application-admintools-ui/src/main/resources/AdminTools/Code/ConfigurationClass.xml

```diff
@@ -84,6 +84,20 @@
       <validationRegExp/>
       <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
     </serverLocation>
+    <spamSize>
+      <customDisplay/>
+      <disabled>0</disabled>
+      <hint/>
+      <name>spamSize</name>
+      <number>3</number>
+      <numberType>long</numberType>
+      <prettyName>spamSize</prettyName>
+      <size>30</size>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <classType>com.xpn.xwiki.objects.classes.NumberClass</classType>
```

Comment:
You set the property to be a NumberClass, so you should easily return an integer rather than a string (from the configuration)